daemon/graphdriver/driver.go

```diff
@@ -46,7 +48,6 @@ type ProtoDriver interface {
 	// String returns a string representation of this driver.
 	String() string
 	// Create creates a new, empty, filesystem layer with the
-	// specified id and parent. Parent may be "".
```

Comment:
Super excited!

Do we know if this is going into 1.9
https://github.com/docker/docker/wiki/Engine-1.9.0 ?


On 12 October 2015 at 14:43, Phil Estes <notifications@github.com> wrote:

> In daemon/graphdriver/driver.go
> <https://github.com/docker/docker/pull/12648#discussion_r41756844>:
>
> > @@ -46,7 +48,6 @@ type ProtoDriver interface {
> >  	// String returns a string representation of this driver.
> >  	String() string
> >  	// Create creates a new, empty, filesystem layer with the
> > -	// specified id and parent. Parent may be "".
>
> Ugh--thanks for catching this @2opremio <https://github.com/2opremio> and
> thanks for quickly correcting it @HuKeping <https://github.com/HuKeping>
> :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12648/files#r41756844>.
>



-- 
- Amos Folarin